tests/testfx.lua

```diff
@@ -244,17 +246,19 @@
 -- Some helper functions
 --
 
-	function test.createsolution()
-		local sln = solution "MySolution"
+	function test.createWorkspace()
+		local wrk = workspace("MySolution")
 		configurations { "Debug", "Release" }
 
-		local prj = project "MyProject"
-		language "C++"
-		kind "ConsoleApp"
+		local prj = project("MyProject")
+		language("C++")
+		kind("ConsoleApp")
```

Comment:
I usually use parens when coding, and no parens when project scripting, if that makes sense. This sort of falls between but I didn't like this:

```lua
local prj = project "prj"
```

…and once I changed that it made since to at least be locally consistent.